tardis/montecarlo/base.py

```diff
@@ -128,6 +129,10 @@ def _initialize_geometry_arrays(self, model):
         self.v_inner_cgs = model.v_inner.to('cm/s').value
 
     def _initialize_packets(self, T, no_of_packets):
+        np.random.seed(self.seed)
+        mc_config_module.packet_seeds = np.random.randint(0,
+                                                          1e9,
```

Comment:
why 1e9?

Reply:
I picked a value that was a few orders of magnitude greater than the largest number of packets I've seen; should I base it instead on `no_of_packets`?